.vscode/launch.json

```diff
@@ -24,9 +24,8 @@
             "name": "Debug Jest Tests",
             "type": "node",
             "request": "launch",
```

Comment:
We should probably undo these changes, as I think I had this working on my computer